.autorc

```diff
@@ -0,0 +1,15 @@
+{
+    "onlyPublishWithReleaseLabel": true,
+    "baseBranch": "maint",
+    "author": "auto <auto@nil>",
```

Comment:
```suggestion
    "author": "DataLad Bot <team@datalad.org>",
```
WDYT @datalad/developers -- which "user" should be  associated with commits adjusting Changelog and the annotated release tags?

Reply:
I guess I will proceed with this and we could adjust later

Reply:
committed locally and pushed